src/user/dto/list-user-by-id.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsString } from 'class-validator';
+
+export class GetUserDTO {
+  @IsNotEmpty({ message: 'O campo não pode ser vazio' })
+  @IsString({ message: 'Não é uma string válida' })
```

Comment:
Se for o mesmo formato o id que o auth0 retorna, você pode criar um regex junto com um matches.

Exemplos com UUID:
```
 @Matches(/^[0-9a-fA-F]{8}\-[0-9a-fA-F]{4}\-4[0-9a-fA-F]{3}\-(8|9|a|b)[0-9a-fA-F]{3}\-[0-9a-fA-F]{12}$/, {
    message: 'O ID do usuário deve ser um UUID válido.',
  })
```

Reply:
Mandou a braba!! Vou fazer aqui agora

Reply:
Mandei bala no regex